documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -0,0 +1,660 @@
+
+|io.debezium.data.Xml
+|n/a
+|`xml`
+|`xml`
+
+|===
+
+In addition to the primitive and logical mappings above, if the source of the change events is a {prodname} source connector, the resolution of the column type, its length, precision, and scale can be further influenced by enabling column or data type propagation.
+This propagation requires that the source connector be configured with `column.propagate.source.type` or `datatype.propagate.source.type` and the {prodname} JDBC sink connector will use these values with higher precedence.
+
+For example, let's say the following field schema is included in a change event:
+
+.{prodname} change event field schema with column/data type propagation enabled
+[source,json]
+----
+{
+  "schema": {
+    "type": "INT8",
+    "parameters": {
+      "__debezium.source.column.type": "TINYINT",
+      "__debezium.source.column.length": "1"
+    }
+  }
+}
+----
+
+In the above example, without the schema parameters, the {prodname} JDBC sink connector would map this field to a column type of `Types.SMALLINT`, which will have varying logical database types depending on the dialect but for MySQL this would be a `TINYINT` column type without any length.
+By enabling the column or data type propagation in the source connector, the {prodname} JDBC sink connector can use this information to refine the data type mapping process and will create the column as `TINYINT(1)`.
+
+[NOTE]
+====
+Typically, column / data type propagation will have a much larger influence when both the source and sink databases are the same.
+We are continually looking at ways to improve this mapping across heterogeneous databases and the current type system allows us to continue to refine these mappings based on feedback.
+If you find a mapping could be improved, please let us know.
+====
+
+[[jdbc-deployment]]
+== Deployment
+
+To deploy a {prodname} JDBC connector, you install the {prodname} JDBC connector archive, configure the connector, and start the connector by adding its configuration to Kafka Connect.
+
+.Prerequisites
+* link:https://zookeeper.apache.org/[Apache ZooKeeper], link:http://kafka.apache.org/[Apache Kafka], and link:{link-kafka-docs}.html#connect[Kafka Connect] are installed.
+* Destiniation database is installed and configured to accept JDBC connections.
+
+.Procedure
+
+. Download the {prodname} https://repo1.maven.org/maven2/io/debezium/debezium-connector-jdbc/{debezium-version}/debezium-connector-oracle-{debezium-version}-plugin.tar.gz[JDBC connector plug-in archive].
+. Extract the files into your Kafka Connect environment.
+. Optionally download the JDBC driver from Maven Central and extract the downloaded driver file to the directory that contains the JDBC sink connector JAR file.
++
+NOTE: Drivers for Oracle and Db2 are not automatically shipped with the JDBC sink connector and must be manually installed.
+. Add the directory with the JAR files to {link-kafka-docs}/#connectconfigs[Kafka Connect's `plugin.path`].
```

Comment:
I am not sure about this wording. We usually don't add plug-in directory to the `plugin.path` but the directory that contains plugin directory.

Reply:
I'll adjust this wording, but this was taken from the Oracle documentation, so @roldanbob we may need to address this in our other connector docs as well and unify the language.